src/test/java/org/opensearch/geospatial/search/aggregations/bucket/geogrid/GeoHexGridTests.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.search.aggregations.bucket.geogrid;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.opensearch.geospatial.GeospatialTestHelper.randomHexGridPrecision;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.lucene.index.IndexWriter;
+import org.hamcrest.MatcherAssert;
+import org.opensearch.common.ParseField;
+import org.opensearch.common.xcontent.ContextParser;
+import org.opensearch.common.xcontent.NamedXContentRegistry;
+import org.opensearch.geo.search.aggregations.bucket.geogrid.BaseGeoGrid;
+import org.opensearch.geo.search.aggregations.bucket.geogrid.BaseGeoGridBucket;
+import org.opensearch.geo.search.aggregations.bucket.geogrid.GeoGrid;
+import org.opensearch.geospatial.h3.H3;
+import org.opensearch.geospatial.plugin.GeospatialPlugin;
+import org.opensearch.plugins.SearchPlugin;
+import org.opensearch.search.aggregations.Aggregation;
+import org.opensearch.search.aggregations.InternalAggregations;
+import org.opensearch.search.aggregations.ParsedMultiBucketAggregation;
+import org.opensearch.test.InternalMultiBucketAggregationTestCase;
+
+// This class is modified from https://github.com/opensearch-project/OpenSearch/blob/main/modules/geo/src/test/java/org/opensearch/geo/search/aggregations/bucket/geogrid/GeoGridTestCase.java
```

Comment:
nit: I think we normally use block style comments for class:
```
/**
 * Class description
 */
```

Reply:
Ack